internal/pkg/pluginmanager/manager.go

```diff
@@ -65,19 +65,16 @@ func DownloadPlugins(conf *configloader.Config) error {
 			return err
 		}
 		// if .so matches with .md5, continue
-		if isMD5Match {
```

Comment:
Perhaps we can simply modify it here. If this code is deleted, then the user will not receive the "plugin exists" message, even if the plugin has actually been downloaded before.

Here is a simple way to implement it: use three conditions in the meanwhile to check if the plugin is been downloaded before.

- condition 1: plugin exist
- condition 2: .md5 exist
- condition 3: isMD5Matc(it is exists in the code now)

### condition 1 :
https://github.com/devstream-io/devstream/blob/ddb7c165300de00d31db97b4c16121cbb7ada2d2/internal/pkg/pluginmanager/manager.go#L37-L50
else condition 1 is true

### condition2:
https://github.com/devstream-io/devstream/blob/ddb7c165300de00d31db97b4c16121cbb7ada2d2/internal/pkg/pluginmanager/manager.go#L51-L61
else condition 2 is true

note: you should not use `contion1` as var name. 🤣